CookieMediator/src/main/java/org/wso2/carbon/connector/creatiocrm/headers/BPMCSRFCookie.java

```diff
@@ -0,0 +1,128 @@
+
+
```

Comment:
Shall we remove these unwanted empty lines?

Reply:
Removed